src/server/api/lib/fakeservice.js

```diff
@@ -28,15 +27,15 @@ async function convertMessagePartsToMessage(messageParts) {
     contactNumber
   })
 
-  const service_id = (firstPart.service_id
+  const serviceId = (firstPart.service_id
                       || `fakeservice_${Math.random().toString(36).replace(/[^a-zA-Z1-9]+/g, '')}`)
   return new Message({
     contact_number: contactNumber,
     user_number: userNumber,
     is_from_contact: true,
     text,
     service_response: JSON.stringify(messageParts),
-    service_id,
+    serviceId,
```

Comment:
You can't change the key of the value sent or it won't be saved to the right field.